src/lib/validators/request-validator.js

```diff
@@ -0,0 +1,21 @@
+import Joi from 'joi';
+import { RequestValidationError } from '../errors';
+
+async function requestValidator(validationRules, request) {
+  const { data } = request;
+
+  if (!validationRules) {
```

Comment:
what's the point of the second check? You already check that it exists in the dispatcher, assume it exists here coz dispatcher is the only place you call this function

Reply:
It's check of existing validationRules for current method, in dispatcher I check existing of fields `validationRules` on controller. I can thow up this check in dispatcher or call requestValidator with controller and method without of validationRules, and in requestValidator get rules for this method.

Reply:
I don't like that both the decorator (that enhancec controllers) and this are called the same, need more precise naming.

Reply:
this is better to be named `contollerValidationRules` then